build.gradle

```diff
@@ -22,7 +22,7 @@ buildscript {
 }
 
 plugins {
-    id 'net.wooga.plugins' version '2.2.0'
+    id 'net.wooga.plugins' version '2.2.2-rc.1'
```

Comment:
I assume this will be updated once that's out.

Reply:
Yeah.